runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -11737,6 +11737,11 @@ J9::X86::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
             return TR::TreeEvaluator::inlineStringLatin1Inflate(node, cg);
             }
          break;
+      case TR::java_util_zip_CRC32C_updateBytes:
+         if (!TR::Compiler->om.canGenerateArraylets())
+            {
```

Comment:
This code needs a capability check for SSE 4.2 I believe (CRC32 instruction availability), unless you check that some other way.

Reply:
Crc32 instruction does not require any feature checks, I will double check all the instructions used though